stdlib/tkinter/messagebox.pyi

```diff
@@ -23,11 +25,87 @@ NO: Final = "no"
 class Message(Dialog):
     command: ClassVar[str]
 
-def showinfo(title: str | None = None, message: str | None = None, **options) -> str: ...
-def showwarning(title: str | None = None, message: str | None = None, **options) -> str: ...
-def showerror(title: str | None = None, message: str | None = None, **options) -> str: ...
-def askquestion(title: str | None = None, message: str | None = None, **options) -> str: ...
-def askokcancel(title: str | None = None, message: str | None = None, **options) -> bool: ...
-def askyesno(title: str | None = None, message: str | None = None, **options) -> bool: ...
-def askyesnocancel(title: str | None = None, message: str | None = None, **options) -> bool | None: ...
-def askretrycancel(title: str | None = None, message: str | None = None, **options) -> bool: ...
+_Icon: TypeAlias = Literal["error", "info", "question", "warning"]
+_Type: TypeAlias = Literal["abortretryignore", "ok", "okcancel", "retrycancel", "yesno", "yesnocancel"]
+_Default: TypeAlias = Literal["abort", "retry", "ignore", "ok", "cancel", "yes", "no"]
```

Comment:
In IDE autocompletions and such, seeing `_Icon` is less helpful than `Literal["error", "info", "question", "warning"]`. I'd suggest just copy/pasting these `Literal`s to all the places that use them instead of type aliases.

An advantage of copy/pasting the Literals is that you can make them function-specific. For example, specifying `default="retry"` doesn't make sense if the dialog only has "yes" and "no" buttons, so  the `default` parameter of `askyesno()` should be `Literal["yes", "no"]`.

There are many existing type aliases in tkinter. For most of them, I would accept PRs that delete them.